### DISCUSSION TITLE
I have one doubt wants to clarify it.

### DISCUSSION THREAD

**Discussion by Musarrath** [jhuckaby/Cronicle#488] [Q&A] [answered]
I am using single server as primary server but if the server crashed and If I wants the server to restart automatically is there any options to automatically restart it or shall I have to use the multiple server only if the primary server crashed then the other server will work as primary server as you mentioned in the documentation.

**Comment by jhuckaby** [comment]
There is no built-in functionality to automatically restart Cronicle if the service crashes.  You could try looking into a process manager like [PM2](https://pm2.keymetrics.io/).

**Comment by dheise-rheinenergie-trading-com** [comment, marked as answer]
My cronicle server are started by the systemd daemon, which has the possibility to restart the cronicle service after a crash.

![image](https://user-images.githubusercontent.com/73586581/163339067-ac97daa7-7c86-4c72-bc5e-f0f71ee2ef12.png)

Works without any hassle.

**Comment by jhuckaby** [comment]
Oh wow, @dheise-rheinenergie-trading-com, I had no idea systemd could do that.  Very cool!  Thank you!  Marking your response as the answer.
